src/DDTrace/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -541,6 +541,22 @@ function (HookData $hook) use ($controllerName, $integration) {
         \DDTrace\trace_method('Symfony\Component\Templating\DelegatingEngine', 'render', $traceRender);
         \DDTrace\trace_method('Symfony\Component\Templating\PhpEngine', 'render', $traceRender);
         \DDTrace\trace_method('Twig\Environment', 'render', $traceRender);
+
+        /* Silence ExecIntegration spans to stty. These are going to fail intentionally,
+         * and always executed within symfony requests. This is pure noise which we hereby silence.
+         */
+        foreach (['Symfony\Component\Console\Terminal::hasSttyAvailable', 'Symfony\Component\Console\Helper\QuestionHelper::isInteractiveInput'] as $method) {
+            \DDTrace\install_hook($method, function (HookData $hook) {
+                \DDTrace\active_stack()->spanCreationObservers[] = function (SpanData $span) use ($hook) {
```

Comment:
```suggestion
                $hook->data = false;
                \DDTrace\active_stack()->spanCreationObservers[] = function (SpanData $span) use ($hook) {
```

Considering that when the pre-hook first runs, `$hook->data` is not initialized, I believe that we would otherwise try to access the value in the span creation closure before it's initialized.

Reply:
That's absolutely right, and thanks for adding tests, hadn't gotten round to that yet!